### PR TITLE
Page type content CMS

### PR DESCRIPTION
**Pull request by tomas-stefano**
With this the videos, news, actions plans, articles will fetch your own content from the CMS like the following:

* Articles will fetch from contento.org.uk/en/articles/slug
* Action Plans will fetch from contento.org.uk/en/action_plans/slug
* News will fetch from contento.org.uk/en/news/slug
* Videos will fetch from contento.org.uk/en/videos/slug

### CODE REVIEW

File: lib/core/interactor/corporate_reader.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Core
+  class CorporateReader < ArticleReader
+    private
```
Comment (tomas-stefano) [inline lib/core/interactor/corporate_reader.rb]: @dougdroper Like we discussed, I inherit from Article Reader. There are some interactors, controllers and views with same concept duplicated. But this refactor will be done in a diff and focused PR.

File: app/controllers/corporate_controller.rb [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class CorporateController < ApplicationController
+  decorates_assigned :corporate, with: ContentItemDecorator
+  include Navigation
+
+  def show
+    @corporate = Core::CorporateReader.new(params[:id]).call do
+      not_found
+    end
+
+    assign_active_categories(*@corporate.categories)
```
Comment (tomas-stefano) [inline app/controllers/corporate_controller.rb]: @dougdroper I was in doubt what categories should I display here in corporate. I put this, at least for now.

**Comment by tomas-stefano** [comment]
Waiting for PR approval :)
waiting-for-pr-approval

**Comment by asmega** [comment]
:+1:

**Comment by dougdroper** [comment]
LGTM :+1:

**Pull request merged**